client/src/bootstrap/sagas.js

```diff
@@ -1,12 +1,13 @@
 import { put, call } from 'redux-saga/effects';
 import { registerPlayer } from '../player/sagas';
-import { createTableAndAddPlayerToTable } from '../table/sagas';
+import { createTableAndAddPlayerToTable, watchOnSnapshot } from '../table/sagas';
 import { setAppLoaded } from '../bootstrap/duck';
 
 export default function* initAppSagas() {
     try {
         yield call(registerPlayer);
         yield call(createTableAndAddPlayerToTable);
+        yield call(watchOnSnapshot);
```

Comment:
I don't think this should be called here at the bootstrap level. IMO, it should be ran each time we "join" a table: we want to be notified of what happpens (and idem, if we were to leave it, we should unsubscribe)
WDYT?

Reply:
Oui, je ne savais vraiment pas où le mettre....